frontend/src/TodoList/TodoItem.tsx

```diff
@@ -1,24 +1,100 @@
 import {Item} from "./model";
+import './TodoItem.css'
+import {useState} from "react";
 
-interface TodoItemProps{
+interface TodoItemProps {
     item: Item;
     onItemDeleted: (items: Array<Item>) => void;
```

Comment:
Think about renaming the function, because it is also called when editing an item.
```suggestion
    onItemListChange: (items: Array<Item>) => void;
```